diff --git a/package/libopenssl/libopenssl.mk b/package/libopenssl/libopenssl.mk
index fc22c204..0bb9db30 100644
--- a/package/libopenssl/libopenssl.mk
+++ b/package/libopenssl/libopenssl.mk
@@ -78,7 +78,9 @@ define LIBOPENSSL_CONFIGURE_CMDS
 			$(if $(BR2_STATIC_LIBS),no-shared,shared) \
 			$(if $(BR2_PACKAGE_HAS_CRYPTODEV),enable-devcryptoeng) \
 			no-rc5 \
-			enable-camellia \
+			no-camellia \
+			no-aria \
+			no-sse2 \
 			no-tests \
 			no-fuzz-libfuzzer \
 			no-fuzz-afl \
