From f0d04007a223353f4f94704b57afc89ab31ba6a4 Mon Sep 17 00:00:00 2001
From: Urja Rannikko <urjaman@gmail.com>
Date: Sat, 30 Mar 2024 11:51:01 +0200
Subject: [PATCH 4/4] smaller libsamplerate

---
 package/libsamplerate/libsamplerate.mk | 1 +
 1 file changed, 1 insertion(+)

diff --git a/package/libsamplerate/libsamplerate.mk b/package/libsamplerate/libsamplerate.mk
index 0ac3c424..aa1a601c 100644
--- a/package/libsamplerate/libsamplerate.mk
+++ b/package/libsamplerate/libsamplerate.mk
@@ -12,6 +12,7 @@ LIBSAMPLERATE_DEPENDENCIES = host-pkgconf
 # sndfile is only used for examples and tests so it doesn't make sense
 # to support it as an optional dependency
 LIBSAMPLERATE_CONF_OPTS = \
+	--disable-sinc-best \
 	--disable-fftw \
 	--disable-sndfile \
 	--program-transform-name=''
-- 
2.44.0

