diff --git a/package/libopenssl/libopenssl.mk b/package/libopenssl/libopenssl.mk
index 487fe55..af86942 100644
--- a/package/libopenssl/libopenssl.mk
+++ b/package/libopenssl/libopenssl.mk
@@ -79,8 +79,10 @@ define LIBOPENSSL_CONFIGURE_CMDS
 			$(if $(BR2_STATIC_LIBS),no-shared,shared) \
 			$(if $(BR2_PACKAGE_HAS_CRYPTODEV),enable-devcryptoeng) \
 			no-rc5 \
-			enable-camellia \
-			enable-mdc2 \
+			no-camellia \
+			no-mdc2 \
+			no-aria \
+			no-sse2 \
 			no-tests \
 			no-fuzz-libfuzzer \
 			no-fuzz-afl \
